index.md

```diff
@@ -100,90 +96,28 @@ the PREMIS digital preservation metadata standard:
-
-One of the basic requirements formulated by the CSIP is the use of METS as the
-metadata standard to describe the structure of an IP.
-
-Special attention was given to the fact that it might not be possible to store
-all representations of an intellectual entity in one physical container, or that
-even a single representation might have to be divided so that it can be stored
-on long-term storage media. For this reason, the AIP format describes means to
-manage representations or representation parts separately.
-
-This structure lays the groundwork for addressing the practical requirement of
-distributing parts of the intellectual entity over a sequence of physical
-containers representing a logical AIP. Even though this puts the integrity of
-the AIP at risk - because in case of disaster recovery the physical container
-does not represent the complete intellectual entity and dependencies to another
-(lost) physical container can potentially make it impossible to interpret,
-understand, or render the content - it is a necessary measure if the amount of
-data exceeds the capacity limitation of long-term storage media.
-
-## Authenticity of the original submission
-
-The AIP format provides a structure for storing the original submission
-separately from any data that is created during SIP-to-AIP conversion and during
-the life-cycle of the AIP. This allows safeguarding the authenticity of the
-original submission.
-
-However, it is an implementation decision if the original submission is kept “as
-is” or if the SIP data is adapted during SIP to AIP conversion. In line with
-OAIS, the content of the original SIP does not have to be identical to the
-version of the submitted data stored as part of the AIP:
-
-> “An OAIS is not always required to retain the information submitted to it in
-> precisely the same format as in the SIP. Indeed, preserving the original
-> information exactly as submitted may not be desirable. [@OAIS2012, p. 4-52]”
-
-The E-ARK AIP format prescribes a structure by defining a set of requirements
-and core metadata together with recommendations on how to use the requirements
-in order to allow changing the AIP while keeping seamless track of the AIP’s
-history.
-
-## Version of an AIP
-
-While the AIP always describes the same unaltered conceptual entity, the way in
-which this information is represented may change. Therefore the AIP format
-describes the means to record the provenance from the time of the first
-submission, and also during the whole life-cycle of the AIP.
-
-For the purpose of the AIP format specification, the *AIP version* concept used
-is as defined by OAIS:
+According to the CSIP, representations are relevant for all types of IPs.
+Regarding the AIP format, in addition to the organization and storage of the 
+representations, it is important that representations which are generated 
+during the AIP's life cycle are managed and documented in such a way that 
+changes can be tracked at any time.
```

Comment:
Not sure as to the meaning of this "that changes can be tracked at any time.". It's the "at any time" thing that's misleading me I think.

Reply:
What was meant is "at any time during the life-cycle". But that's obvious, I removed it.